From bd02f316e125b344918dd21a3c57c166032a8994 Mon Sep 17 00:00:00 2001
From: Rong Dian <b38775@freescale.com>
Date: Tue, 10 Jul 2012 19:23:50 +0800
Subject: ENGR00216327: battery: fail to update battery voltage with usb
 charger attached

when usb charger is online, driver also updates battery voltage

Signed-off-by: Rong Dian <b38775@freescale.com>
---
 drivers/power/sabresd_battery.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/power/sabresd_battery.c b/drivers/power/sabresd_battery.c
index 94d7e41..1e13ebf 100755
--- a/drivers/power/sabresd_battery.c
+++ b/drivers/power/sabresd_battery.c
@@ -265,7 +265,7 @@ static void max8903_battery_update_status(struct max8903_data *data)
 		data->voltage_uV = temp_last;
 		}
 	}
-	if (data->charger_online == 1) {
+	if (data->charger_online == 1 || data->usb_charger_online == 1) {
 		data->voltage_uV = temp;
 		temp_last = temp;
 	}
-- 
1.7.9.5

